src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -118,5 +145,8 @@ public static ProviderCultureResult ParseCookieValue(string value)
 
             return new ProviderCultureResult(cultureName, uiCultureName);
         }
+
+        private static bool IsValidCulture(string cultureName) =>
+            CultureInfo.GetCultures(CultureTypes.SpecificCultures).Any(c => c.Name == cultureName);
```

Comment:
I think this check needs to be an *identical* check to whatever prompted this change in the first place. That is, when there's an invalid culture, what call used to fail? Let's just do that call, no?